File: examples/simp/CPP/evaluator/evaluator5.cpp
#include <linkpred.hpp>
#include <iostream>

using namespace LinkPred::Simp;

int main() {
	double edgeRemRatio = 0.1;
	bool keepConnected = false;
	long int seed = 0;
	Evaluator eval;
	eval.genTestData("Zakarays_Karate_Club.edges", "Zakarays_Train.edges",
			"Zakarays_Test.edges", edgeRemRatio, keepConnected, seed);
	return 0;
}

File: examples/simp/Python/evaluator/evaluator5.py
# Import the library
import LinkPredPython as lpp
edgeRemRatio = 0.1;
keepConnected = False;
seed = 0;
# Create an evaluator object
ev = lpp.Evaluator();
ev.genTestData("Zakarays_Karate_Club.edges", "Zakarays_Train.edges", "Zakarays_Test.edges", edgeRemRatio, keepConnected, seed);
